build/yamls/base/agent-rbac.yml

```diff
@@ -149,6 +149,24 @@ rules:
       - get
       - watch
       - list
+  - apiGroups:
+      - crd.antrea.io
+    resources:
+      - egresses/status
+    verbs:
+      - get
+      - watch
+      - list
+      - update
```

Comment:
I think only `update` is required for egresses/status?

Reply:
yes, only update is required
```yaml
  - apiGroups:
      - crd.antrea.io
    resources:
      - egresses/status
    verbs:
      - update
```